enterprise/server/auth/auth.go

```diff
@@ -17,10 +16,12 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/tables"
 	"github.com/buildbuddy-io/buildbuddy/server/util/capabilities"
 	"github.com/buildbuddy-io/buildbuddy/server/util/db"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
 	"github.com/buildbuddy-io/buildbuddy/server/util/lru"
 	"github.com/buildbuddy-io/buildbuddy/server/util/random"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 
+	"github.com/coreos/go-oidc"
```

Comment:
this makes it unclear where oidc comes from (go-oidc has package "oidc") which is why I aliased it initially

Reply:
Done.